backend/app.py

```diff
@@ -257,3 +263,7 @@ def refresh_model():
     msg = 'New model improved accuracy from {} to {}'.format(score_before, score_after)
     return Response(body=msg, status_code=201)
 
+
+def get_current_user_email():
+    req_context = app.current_request.context
```

Comment:
what is inside `current_request`?

Reply:
That's provided by the chalice framework. Yes, with the `authorizer` instrumented, it will look up the user info and put it in the context.